proto/context/src/main/proto/spine/validation/field_views.proto

```diff
@@ -11,17 +11,30 @@ option java_multiple_files = true;
 
 import "spine/protodata/ast.proto";
 
+// Uniquely identifies a message field.
```

Comment:
I would say it uniquely identifies a declaration of a field in some `Message`. 

It doesn't identify the field instance itself, as one might think reading this.

Reply:
>> It doesn't identify the field instance itself, as one might think reading this.

It does because it has both the parent type name and the field name.